README.md

```diff
@@ -513,7 +538,8 @@ Offline Installations   | ✓     | x   | x   |
   Manifest JSON document that describes the application's dependencies and sources.
 * **Dependency Replacements** - Dependency replacements can be specified when creating a Cachito
   request. This is a convenient feature to allow dependencies to be swapped without making changes
-  in the source repository.
+  in the source repository. Dependency replacement is only supported if a single module is referenced
```

Comment:
```suggestion
  in the source repository. Dependency replacement is only supported if a single package is referenced
```
I know module is the more correct golang term here, but we've been using package everywhere